### PR TITLE
Sr at21 integration test config8

### PR DESCRIPTION
**Pull request by simen-rekkedal**
## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/wwwroot/authentication/nothing.cs
Comment (acn-dgopa) [inline bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/wwwroot/authentication/nothing.cs:1]: I dont think you need a nothing.cs but a gitignore file must be sufficient

File: bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Altinn.Authentication.UI.csproj [hunk 1]
```diff
@@ -31,8 +31,4 @@
     <ProjectReference Include="..\Altinn.Authentication.UI.Mocks\Altinn.Authentication.UI.Mocks.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
-    <Folder Include="wwwroot\authentication\" />
-  </ItemGroup>
```
Comment (acn-dgopa) [inline bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Altinn.Authentication.UI.csproj:36]: but why is this removed?

Comment (simen-rekkedal) [reply bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Altinn.Authentication.UI.csproj:36]: trengs ikke når vi har en .gitkeep fil

**Review by acn-dgopa — APPROVED** [review]
